system_baseline/models.py

```diff
@@ -48,7 +48,15 @@ def validate_facts(self, key, value):
         validators.check_for_duplicate_names(value)
         return value
 
-    def mapped_system_ids(self, rbac_group_filters=None):
+    def get_query_from_filters(self, filters):
```

Comment:
Name suggestion: `get_groups_query_filter`

Reply:
Naming is hard...